modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFMosaicReaderTest.java

```diff
@@ -1916,6 +1916,7 @@ public void testGranuleSourceFileView() throws Exception {
                 nc2 = it.next();
             }
 
+            TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
Good call, but done like this could be source of trouble: the test JVM is re-used across modules, meaning other tests might get this same timezone while before they did not. Also, we use parallel builds, which other modules tests might be affected by this would be semi-random (depending on the performance of the hardware, how many parallel processes are used for the build, and so on).

Best to isolate this properly, like done in this test instead:
https://github.com/geotools/geotools/blob/main/modules/plugin/geopkg/src/test/java/org/geotools/geopkg/JSONArrayIOTest.java#L35